config/locales/cy.yml

```diff
@@ -86,3 +86,5 @@ cy:
           pension_calculator_html: Defnyddiwch ein <a href="#">Cyfrifiannell Pensiwn</a> i weld faint o bensiwn y bydd eich cronfa yn ei chronni dros amser.
           workplace_pensions_html: Dysgwch ragor am <a href="#">bensiynau gweithle</a>.
           budget_planner_html: Defnyddiwch ein <a href="#">cynllunydd cyllideb</a> i weld pa effaith y bydd eich cyfraniadau yn eu cael ar eich incwm.
+      edit_frequency:
+        label: Change salary frequency
```

Comment:
@margOnline shall we add a clear note to indicate that this content shall be altered at a more convenient time i.e. when the translation is available?

`label: Change salary frequency` --> `label: Change salary frequency` TODO: Awaiting Translation